docs/source/tutorial_chat_service.md

```diff
@@ -11,13 +11,15 @@ We currently support the following chat services:
 
 You can find more information on how to set up these services below.
 
-Please read here [here](https://github.com/facebookresearch/ParlAI/tree/master/parlai/chat_service) for information on how to set up a new chat service.
+:::{note}
+If you'd like to use a service outside of the 4 listed above, please read [here](https://github.com/facebookresearch/ParlAI/tree/master/parlai/chat_service) for information on how to set up a new chat service.
+:::
 
 ## Overview
 
-As stated, humans messaging on chat services can be viewed as a type of agent in ParlAI, communicating with models via observations and action dicts. See [here](https://github.com/facebookresearch/ParlAI/blob/master/parlai/chat_service/core/agents.py) for the basic implementation. Human agents are placed in worlds with ParlAI agent(s) and possibly other humans, and the world defines how each of these agents interacts.
+As stated, humans messaging on chat services can be viewed as a type of agent in ParlAI, communicating with models via observations and action dicts. You can find the implementation of `ChatServiceAgent`s [here](https://github.com/facebookresearch/ParlAI/blob/master/parlai/chat_service/core/agents.py). Human agents are placed in worlds with ParlAI agent(s) and possibly other humans, and the world defines how each of these agents interacts.
```

Comment:
do you want chat_service to be part of the API reference? See what's done in api.md and the core folder if you want.

Reply:
i can't find any `api.md` file?

Reply:
ah sorry it's core.md